lib/ts/recipe/multitenancy/types.ts

```diff
@@ -173,6 +173,9 @@ export type APIInterface = {
               passwordless: {
                   enabled: boolean;
               };
+              webauthn: {
+                  credentialIds: string[];
+              };
```

Comment:
no need for this. There is already a firstFactors array output which will contain webauthn id if its enabled as first factor.

Reply:
done